apps/charterafrica/src/components/OpportunityCardList/OpportunityCardList.js

```diff
@@ -0,0 +1,132 @@
+import { Typography, Box, Grid, Button } from "@mui/material";
+import React, { useState } from "react";
+
+import OpportunityCard from "../OpportunityCard";
+
+import { neutral } from "@/charterafrica/colors";
+
+const OpportunityCardList = React.forwardRef(function OpportunityCardList(
+  props,
+  ref
+) {
+  const { title, grants } = props;
+
+  const [showAll, setShowAll] = useState(false);
+
+  if (!grants) {
+    return null;
+  }
+  return (
+    <Box ref={ref}>
+      <Grid container justifyContent="space-between" mb={5}>
+        <Grid item xs={12} md={6}>
+          <Typography
+            color={neutral[900]}
+            textAlign={{ xs: "center", sm: "left" }}
+            textTransform="capitalize"
+            variant="h5SemiBold"
+          >
+            {title}
```

Comment:
Check the font weight on desktop

Reply:
@koechkevin I can see that its `400` across all of the